js/fullWidthTreeView.js

```diff
@@ -218,12 +235,39 @@
     // a node is hovered to make it appear after node changes. It must
     // use the #fullwidth-treeview container to allow a higher
     // height than the node in multiple lines tooltips
+    var syncedParents = {};
     var hoverNodeListener = function (e, data)
     {
+      // Do nothing if node is disabled
+      if ($fwTreeView.jstree("is_disabled", data.node.id)) {
+        return;
+      }
+
+      // Configure tooltip
       $('a.jstree-anchor').tooltip({
         delay: 250,
         container: '#fullwidth-treeview'
       });
+
+      // Check sync of treeview data
+      var parent = data.node.parent;
+      var parentNode = $fwTreeView.jstree("get_json", parent);
+
+      if (parent != "#" && !(parent in syncedParents) && ("href" in parentNode.a_attr)) {
+        syncedParents[parent] = true;
```

Comment:
I'd set this only on success, when the node is being refreshed, that way it could be retried if the repair didn't work.